repos/system_upgrade/common/actors/scanpkgmanager/libraries/scanpkgmanager.py

```diff
@@ -28,5 +33,53 @@ def get_etc_releasever():
     return releasever
 
 
+def get_config_contents(config_path):
```

Comment:
Nit: If it's a helper designed for a one time use in `get_configured_proxies` function I would rename the function to `_get_config_contents` to make it clear